src/ui/select/select.component.tsx

```diff
@@ -268,9 +268,8 @@ const Select = React.forwardRef<SelectRef, SelectProps>(
       );
     };
 
-    const renderNoResultsFound = () => {
+    const renderNoResultsFound = () =>
```

Comment:
@jsartisan , I'm curious why this isn't passed in as the `ListEmptyComponent` prop of `FlatList` instead of conditionally rendering based on `filteredData.length`.